lib/schema.ex

```diff
@@ -97,12 +97,10 @@ defmodule Rajska.Schema do
 
   defp validate_scope!(false, _role, _authorization), do: :ok
 
-  defp validate_scope!(:source,  _role, _authorization), do: :ok
-
   defp validate_scope!(scope, _role, _authorization) when is_atom(scope) do
-    scope.__schema__(:source)
+    struct!(scope)
   rescue
-    UndefinedFunctionError -> reraise ":scope option #{inspect(scope)} doesn't implement a __schema__(:source) function.", __STACKTRACE__
+    UndefinedFunctionError -> reraise ":scope option #{inspect(scope)} is not a struct.", __STACKTRACE__
```

Comment:
```suggestion
    FunctionClauseError -> reraise ":scope option #{inspect(scope)} is not a struct.", __STACKTRACE__
```

Reply:
This makes the tests fail, because `struct!` actually throws an UndefinedFunctionError:

`Expected exception RuntimeError but got UndefinedFunctionError (function Rajska.SchemaTest.NotStruct.__struct__/1 is undefined or private)`